gems/ibm_cloud_power/lib/ibm_cloud_power/models/sap_profile.rb

```diff
@@ -149,24 +149,12 @@ def list_invalid_properties
     def valid?
       return false if @profile_id.nil?
       return false if @type.nil?
-      type_validator = EnumAttributeValidator.new('String', ["balanced", "compute", "memory", "non-production", "ultra-memory"])
-      return false unless type_validator.valid?(@type)
       return false if @cores.nil?
       return false if @memory.nil?
       return false if @certified.nil?
       true
     end
 
-    # Custom attribute writer method checking allowed values (enum).
-    # @param [Object] type Object to be assigned
-    def type=(type)
```

Comment:
Should we be deleting `type=` entirely?  Think we should just drop the validator and set `@type = type` (which is really just the same as `attr_writer :type`